blockchain/blockchain-wallet-v4-frontend
# fix(Send BTC): custom fee ux

## Description
793

## Change Type
Please enter one or more of the following: 
- Bug Fix
- Upgrades

## Testing Steps
1. Open send
2. Go to 'customize fee'
3. Enter 0 sat/byte, see required error
4. Enter 1 sat/byte, see warning, but can still send
5. Click warning, don't see error

## Code Checklist
